pkgs/top-level/perl-packages.nix

```diff
@@ -14028,6 +14421,22 @@ let
     };
   };
 
+  MongoDB = buildPerlPackage {
```

Comment:
Maybe worth skipping this?  If upstream itself won't support it anymore, I don't think it should be here.